src/zampy/datasets/eth_canopy_height.py

```diff
@@ -71,15 +80,16 @@ def download(  # noqa: PLR0913
                 overwrite=overwrite,
             )
         return True
-    
-    def raw_load(
+
+    def preprocess(
         self,
         download_dir: Path,
         spatial_bounds: SpatialBounds,
-        temporal_bounds: Tuple[np.datetime64, np.datetime64],
-        variables: List[Variable],
-    ):
-        pass
+        time_bounds: TimeBounds,
+        variable_names: List[str],
+    ) -> bool:
+        """Preprocess the downloaded data to the CF-like Zampy convention."""
+        return False
```

Comment:
Some thought for later. Maybe we'd better use another name for this function. "preprocess" sounds like a word for operations like regridding, etc. To convert raw data to our generic zampy convention, we can create a hidden function with another name. And this hidden function can be triggered when we load data into memory (or during the conversion to other convention). Then it is not needed for the user to call this extra step.

Reply:
It's a good point, I've chosen to name this step "ingest" now (#7 ).

In general, I don't think users will directly interact with the methods anyway: interaction is done through recipes.